include/boost/geometry/algorithms/detail/assign_indexed_point.hpp

```diff
@@ -49,10 +57,13 @@ inline void assign_point_to_index(Point const& point, Geometry& geometry)
     concepts::check<Point const>();
     concepts::check<Geometry>();
 
-    detail::assign::assign_point_to_index
-        <
-            Geometry, Point, Index, 0, dimension<Geometry>::type::value
-        >::apply(point, geometry);
+    detail::for_each_dimension<Geometry>([&](auto dimension){
```

Comment:
In almost all code you have the curly brace on next line in these lambda's

Reply:
Done.